src/molecules/Popover/Popover.styled.ts

```diff
@@ -0,0 +1,68 @@
+import styled from "styled-components";
+import { colorsPalette } from "styles";
+
+export const POPOVER_PADDING = "16px";
+export const POPOVER_MAX_HEIGHT = "70vh";
+export const POPOVER_HEADER_HEIGHT = "48px";
+
+export const PopoverContainer = styled.div`
+  background: #ffffff;
+  box-shadow: 0px 0px 33px 6px rgba(66, 68, 90, 0.35);
+  border-radius: 8px;
+  width: fit-content;
+  max-width: 300px;
```

Comment:
maybe it is worth to add more width here? like 350 would still be good and will able to hold more text